.nvmrc

```diff
@@ -1 +0,0 @@
-10.17.0
```

Comment:
Could we bump it to 12 instead of removing it?
Me and nulltoken use it.

Reply:
Done!